src/mode/modeHandler.ts

```diff
@@ -129,50 +129,63 @@ export class VimSettings {
   useCtrlKeys     = false;
 }
 
+export enum SearchDirection {
```

Comment:
Ah, I see. Hmm, if we're using the values of an enum directly, I'm not sure if this change actually improves readability, since that is not really standard. What do you think?